UploadTaskNativeModule.js

```diff
@@ -1,18 +1,35 @@
 //  Created by react-native-create-bridge
```

Comment:
Ok, so this file is (was) our native module JS file. We should be able to remove this file and use `react-native-background-upload`'s `Upload` directly. You're basically just wrapping in here.

Reply:
I see you imported it in `TextileSagas.js` like you were going to use it, but I don't see any other change in that file.